circle.yml

```diff
@@ -1,11 +1,10 @@
 machine:
   node:
-    version: 0.12.0
+    version: 6.11.3
```

Comment:
Should these docs which currently say `6.9.0` be updated?
https://github.com/ripple/ripple-lib/pull/760